compiler/compiler_test.go

```diff
@@ -73,44 +384,116 @@ type source struct {
 	contents []byte
 }
 
-func compile(path string, sourceFiles []source, minify bool) ([]byte, error) {
-	conf := loader.Config{}
-	conf.Fset = token.NewFileSet()
-	conf.ParserMode = parser.ParseComments
+func compile(t *testing.T, sourceFiles []source, minify bool) []byte {
```

Comment:
Some of these helper methods seem to be general enough to be useful for testing other packages. Maybe move them into `internal/srctesting`?

Reply:
I won't be able to move any of them other than `ParseSources` without making `srctesting` depend on `compiler` because of the `Archive`'s. We could move more by using `package *_test` tests but for now I'll just `ParseSources`, if that's okay.

Reply:
That's a good point. Let's keep it as is then.